backend/services/services.py

```
@@ -0,0 +1,213 @@
+    return f"{user_id}_{secrets.token_hex(8)}{file_ext}"
+
+
+def create_post_entry(db: Session, user_id: int, content: Optional[str], post_type: str) -> Post:
+    post = Post(content=content, user_id=user_id, post_type=post_type)
+    db.add(post)
+    db.commit()
+    db.refresh(post)
+    return post
+
+
+def send_post_notifications(db: Session, user: User, post: Post):
```

Ruff (0.8.2)

Missing return type annotation for public function `send_post_notifications`

Add return type annotation: `None`

(ANN201)